Src/FluentAssertions.DataSets/FluentAssertions.DataSets.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <PropertyGroup Label="Package info">
-    <Authors>Dennis Doomen;Jonas Nyrup</Authors>
+    <Authors>Dennis Doomen;Jonathan Gilbert</Authors>
```

Comment:
A few lines below
`<RepositoryUrl>https://github.com/fluentassertions/fluentassertions.data</RepositoryUrl>` change to
`<RepositoryUrl>https://github.com/fluentassertions/fluentassertions.datasets</RepositoryUrl>`

Reply:
And grep'ing for `fluentassertions\.data[^s]`
```
AcceptApiChanges.sh:6:find Tests/Approval.Tests/ApprovedApi/FluentAssertions.Data/ -type f -name "*received.txt" | perl -pe 'print $_; s/received/verified/' | xargs -n2 mv
README.md:11:[![GitHub commit activity](https://img.shields.io/github/commit-activity/m/fluentassertions/fluentassertions.datasets)](https://github.com/fluentassertions/fluentassertions.data/graphs/commit-activity)
```